src/bitops.c

```diff
@@ -198,6 +246,12 @@ long long serverPopcount(void *s, long count) {
         return popcountAVX2(s, count);
     }
 #endif
+#ifdef __ARM_NEON
```

Comment:
The other code is guarded with the `__aarch64__` macro, so we should probably use the same here here. I guess there is som 32-bit ARM where `__ARM_NEON` is defined.

```suggestion
#ifdef __aarch64__
```